src/ps2.h

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#ifndef PS2_H
+#define PS2_H
+
+#define PS2_DATA_PORT 0x60
+#define PS2_STATUS_REGISTER 0x64
+#define PS2_COMMAND_REGISTER 0x64
+#define PS2_TIMEOUT 1
+
+#include "ioports.h"
+
+void ps2_init();
```

Comment:
probably not. if mouse_init works without using ps2_init, we can probably ignore the tests and initialization code in ps2_init